frontend/src/app/features/book/components/series-page/series-page.component.html

```
@@ -261,15 +261,15 @@ <h2 class="series-title">{{ seriesTitle() }}</h2>
                     <span>{{ t('synopsis') }}</span>
                   </div>
                   <div class="description-body">
-                    <div #descriptionContent [ngClass]="{ 'line-clamp-7': !isExpanded }"
+                    <div #descriptionContent [ngClass]="{ 'line-clamp-7': !isExpanded() }"
                       class="description-content">
                       <div class="description-html"
                         [innerHTML]="firstDescription() || t('noDescription')"></div>
                     </div>
-                    @if (isExpanded || isOverflowing) {
+                    @if (isExpanded() || isOverflowing()) {
                       <button class="expand-toggle" (click)="toggleExpand()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)